src/Objs/Job/JobEditingConfig.js

```diff
@@ -90,4 +90,28 @@ Scrivito.provideEditingConfig("Job", {
     ...metadataInitialContent,
     body: [new SectionWidget({})],
   },
+  validations: [
+    [
+      "title",
+
+      title => {
+        if (!title) {
+          return {
+            message: "Job title must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    obj => {
```

Comment:
Please mark this validation for either the attribute `datePosted` or the attribute `validThrough`. I prefer the later option.

See "Attribute validation using callbacks" under https://www.scrivito.com/js-sdk/provideEditingConfig#validations for details.

This enables Scrivito to provide an on click action for the given message.